pkg/httpproxy/sign.go

```diff
@@ -19,12 +19,15 @@ import (
 	"github.com/rancher/types/apis/core/v1"
 )
 
+const (
+	ConnectionHeader    = "connection"
```

Comment:
@vincent99 can you think of any other headers that might need to get dropped before signing the request?